docs/ai-providers/aws-bedrock.md

```
@@ -0,0 +1,175 @@
+1. **Install boto3**: AWS Bedrock requires boto3 version 1.28.57 or higher:
+   ```bash
+   pip install "boto3>=1.28.57"
+   ```
+
+2. **AWS credentials**: Ensure you have AWS credentials configured with access to Bedrock models. See [AWS Docs](https://docs.aws.amazon.com/bedrock/latest/userguide/getting-started.html){:target="_blank"}.
+
+## Configuration
+
+=== "Holmes CLI"
+
+    ```bash
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)